### PR TITLE
fix: set prefix to localStorage

### LINKED ISSUE #5556: Refreshing page on Kusama will change it to Basilisk
Refreshed our page and it always changes chain to basilisk. It's annoying. 

https://user-images.githubusercontent.com/104380776/230653632-ad3a0bab-9661-4aa9-a001-218bc451a4ee.mov

### LINKED ISSUE #5576: Remember to be connected user
Hey I think we need for users to connect again even I was connected in previous tab? Is it on purpose?
Is it possible to keep user logged in?

https://user-images.githubusercontent.com/5887929/230786349-ff331dd3-9381-425d-bb79-df433cb1730c.mov

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5576
- [x] Closes #5556

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by vikiival — APPROVED** [review]

File: composables/usePrefix.ts [hunk 1]
```diff
@@ -3,9 +3,18 @@ import { getKusamaAssetId } from '@/utils/api/bsx/query'
 export default function () {
   const { $store } = useNuxtApp()
   const route = useRoute()
+  const storage = useLocalStorage('urlPrefix', { selected: 'bsx' })
```
Comment (roiLeo) [inline composables/usePrefix.ts:6]: maybe set 'bsx' as constant (`DEFAULT_PREFIX`)

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by yangwao** [comment]
I think it's for separate issue? 

When I switch to "RMRK2" it lands this

https://user-images.githubusercontent.com/5887929/231151663-6ce4ef70-8f3f-4ac3-83fb-7f37759226d1.mov

**Review by roiLeo — CHANGES_REQUESTED** [review]
Hey we still have a small issue mayberelated with https://github.com/kodadot/nft-gallery/pull/5472#discussion_r1157003027, I can't manage to change network from explore menu

**Review by roiLeo — APPROVED** [review]
✅ wfm

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 33.29 USD/KSM ~ 0.901 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x26e68d24f0da3afae1e8d6c4f9408b2aa4e05e0e1175f177cddc7776d0fff8e4](https://kusama.subscan.io/extrinsic/0x26e68d24f0da3afae1e8d6c4f9408b2aa4e05e0e1175f177cddc7776d0fff8e4)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**